WooCommerce/Classes/ViewRelated/Dashboard/Onboarding/StoreOnboardingPaymentsSetupView.swift

```diff
@@ -0,0 +1,207 @@
+import SwiftUI
+
+/// Hosting controller that wraps the `StoreOnboardingPaymentsSetupView`.
+final class StoreOnboardingPaymentsSetupHostingController: UIHostingController<StoreOnboardingPaymentsSetupView> {
+    init(task: StoreOnboardingPaymentsSetupCoordinator.Task,
+         onContinue: @escaping () -> Void,
+         onDismiss: @escaping () -> Void) {
+        super.init(rootView: StoreOnboardingPaymentsSetupView(task: task,
+                                                              onContinue: onContinue,
+                                                              onDismiss: onDismiss))
+    }
+
+    @available(*, unavailable)
+    required dynamic init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+/// Shows an overview of the payments setup with a CTA to set up payments in a webview.
+struct StoreOnboardingPaymentsSetupView: View {
+    typealias Task = StoreOnboardingPaymentsSetupCoordinator.Task
+
+    // Tracks the scale of the view due to accessibility changes.
+    @ScaledMetric private var scale: CGFloat = 1.0
+    @State private var urlToOpen: URL?
+
+    private let task: Task
+    private let onContinue: () -> Void
+    private let onDismiss: () -> Void
+
+    init(task: Task,
+         onContinue: @escaping () -> Void,
+         onDismiss: @escaping () -> Void) {
+        self.task = task
+        self.onContinue = onContinue
+        self.onDismiss = onDismiss
+    }
+
+    var body: some View {
+        ScrollView {
+            VStack(alignment: .leading, spacing: Layout.verticalSpacing) {
+                HStack {
+                    Circle()
+                        .fill(Color(.wooCommercePurple(.shade0)))
+                        .frame(width: Layout.circleDimension * scale, height: Layout.circleDimension * scale)
+                        .overlay {
+                            Image(uiImage: .currencyImage)
```

Comment:
I also noticed this and made sure our asset is preserving vector content and using a PDF. This seems to be a SwiftUI issue that requires [some non-ideal workaround](https://stackoverflow.com/a/61178828/9185596) at the moment. I updated the asset since the image size is about the same: https://github.com/woocommerce/woocommerce-ios/pull/9188/commits/88e267c055622953d74ed1374e6618ed6b307bee

Also did a regression test on the other use case (the currency icon in the list item) to make sure there are no changes:

before | after
-- | --
<img width="828" alt="Screenshot 2023-03-20 at 9 44 08 AM" src="https://user-images.githubusercontent.com/1945542/226229434-cff17906-747e-4b2b-92d4-e94f20255810.png"> | <img width="828" alt="Screenshot 2023-03-20 at 9 47 20 AM" src="https://user-images.githubusercontent.com/1945542/226229440-27cbc23a-15fe-42fc-8576-003ef86cd8f3.png">